algorithms/unit_tests/Makefile

```diff
@@ -18,6 +18,8 @@ LINK ?= $(CXX)
 LDFLAGS ?=
 override LDFLAGS += -lpthread
 
+KOKKOS_USE_DEPRECATED_MAKEFILES=1
```

Comment:
Why are we editing these files?
Isn't the idea that this variable gets set int the environment?

Reply:
It makes more sense to me that users would modify their Makefiles accordingly but `make` also imports environment variables.

Reply:
Then why do we insert that definition in any of the makefiles and not just require that being set in the environment?

Reply:
Yes a project that can't switch yet should set this variable during the transition period, and not need people checking out their code to set an environment variable for stuff to work.